builders/container-apps-internal-registry-demo/buildCappsBuilder.sh

```diff
@@ -12,7 +12,6 @@ source $REPO_DIR/build/__variables.sh
 destinationFqdn="oryxprodmcr.azurecr.io"
 destinationRepo="public/oryx/builder"
 destinationTag="capps-20230208.1"
-baseBuilderImage="mcr.microsoft.com/oryx/builder:debian-bullseye-20231016.1"
```

Comment:
Could you please share more details about this part of the change?

Reply:
This just makes it so that I don't need to update the builder image in 2 places. So running this script will use whatever builder is defined in the Dockerfile, rather than overriding it.